src/components/ConfigureRowColSlots.tsx

```diff
@@ -27,7 +25,8 @@ interface IDisplayConfigurableSlotsProps {
 
 export const ConfigureRowColSlots: React.FC<{
   selectedBuildingFloorSlots: IConfigureRowColSlots;
-}> = ({ selectedBuildingFloorSlots }) =>
+  onSubmission: (db: IParkingSlotsDB[][]) => void; // New prop
```

Comment:
Can we have a relevant comment of why onSubmission is needed here?

Reply:
Commented